testresults/src/org/labkey/testresults/view/failureDetail.jsp

```diff
@@ -28,81 +25,66 @@
-    String viewType = data.getViewType();
     String failedTest = getViewContext().getRequest().getParameter("failedTest");
     Container c = getContainer();
     RunDetail[] runs = data.getStatRuns();
-    if(runs.length > 1) {
-        Arrays.sort(runs, new Comparator<RunDetail>()
-        {
-            @Override
-            public int compare(RunDetail o1, RunDetail o2)
-            {
-                return o2.getRevision() - o1.getRevision();
-            }
-        });
+    if (runs.length > 1) {
+        Arrays.sort(runs, (o1, o2) -> o2.getRevision() - o1.getRevision());
     }
     Map<String, List<TestFailDetail>> languageFailures = new TreeMap<>();
     languageFailures.put(failedTest, Arrays.asList(data.getFailures()));
     Map<String, Map<String, Double>> languageBreakdown = data.getLanguageBreakdown(languageFailures);
 
     DateFormat df = new SimpleDateFormat("MM/dd/YYYY HH:mm");
     DateFormat dfEnd = new SimpleDateFormat("MM/dd/YYYY");
+    DateFormat jsDf = new SimpleDateFormat("yyyy-MM-dd");
 %>
 
-<div id="menu">
-    <ul>
-        <li><a href="<%=h(new ActionURL(TestResultsController.BeginAction.class, c))%>" style="color:#fff;">-Overview</a></li>
-        <li><a href="<%=h(new ActionURL(TestResultsController.ShowUserAction.class, c))%>" style="color:#fff;">-User</a></li>
-        <li><a href="<%=h(new ActionURL(TestResultsController.ShowRunAction.class, c))%>" style="color:#fff;">-Run</a></li>
-        <li><a href="<%=h(new ActionURL(TestResultsController.LongTermAction.class, c))%>" style="color:#fff;">-Long Term</a></li>
-        <li><a href="<%=h(new ActionURL(TestResultsController.ShowFlaggedAction.class, c))%>" style="color:#fff;">-Flags</a></li>
-        <li><a href="<%=h(new ActionURL(TestResultsController.TrainingDataViewAction.class, c))%>" style="color:#fff;">-Training Data</a></li>
-        <li><a href="<%=h(new ActionURL(TestResultsController.ErrorFilesAction.class, c))%>" style="color:#fff;">-Posting Errors</a></li>
-        <li><a href="https://skyline.gs.washington.edu/labkey/project/home/issues/begin.view?" target="_blank" title="Report bugs/Request features.  Use 'TestResults' as area when creating new issue" style="color:#fff;">-Issues</a></li>
-        <img src="<%=h(contextPath)%>/TestResults/img/uw.png" id="uw">
-    </ul>
-</div>
+<%@include file="menu.jsp" %>
 <script type="text/javascript">
     LABKEY.requiresCss("/TestResults/css/style.css");
     LABKEY.requiresCss("/TestResults/css/tablesorter-default.css");
 </script>
-<script src="<%=h(contextPath)%>/TestResults/js/d3.v3.js"></script>
+<link rel="stylesheet" href="<%=h(contextPath)%>/TestResults/css/c3.min.css">
+<script src="<%=h(contextPath)%>/TestResults/js/d3.min.js"></script>
 <script src="<%=h(contextPath)%>/TestResults/js/c3.min.js"></script>
 <script src="//code.jquery.com/jquery-1.10.2.js"></script>
 <script src="<%=h(contextPath)%>/TestResults/js/jquery.tablesorter.js"></script>
 <br />
+<%
```

Comment:
These should be declared as client dependencies.  See the addClientDependencies() override in qcSummary.jsp in the targetedms module.
Add dependency on the LabKey internal JQuery version.  Make sure that your code still works with this version.
dependencies.add("internal/jQuery")